exercises/etl/etl_test.rb

```diff
@@ -1,27 +1,36 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'etl'
 
-class TransformTest < Minitest::Test
-  def test_transform_one_value
+# Test data version: 040a24f
+class EtlTest < Minitest::Test
+  def test_a_single_letter
+    # skip
     old = { 1 => ['A'] }
     expected = { 'a' => 1 }
 
     assert_equal expected, ETL.transform(old)
   end
 
-  def test_transform_more_values
+  def test_single_score_with_multiple_letters
     skip
-    old = { 1 => %w(A E I O U) }
-    expected = { 'a' => 1, 'e' => 1, 'i' => 1, 'o' => 1, 'u' => 1 }
```

Comment:
The previous test was arbitrarily placing key/value pairs on newlines.
I decided it was easiest to just check if there was more than one key/value pair in a hash 
(by matching on ',') and if so, then printing each key value pair on a separate line.